packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx

```diff
@@ -42,10 +42,19 @@ export const InviteMembersForm = (props: InviteMembersFormProps) => {
     label: localizationKeys('formFieldLabel__emailAddresses'),
   });
 
+  const defaultRole = useDefaultRole();
   const roleField = useFormControl('role', '', {
     label: localizationKeys('formFieldLabel__role'),
   });
 
+  useEffect(() => {
+    if (roleField.value || !defaultRole) {
```

Comment:
Not sure if that's the expected pattern on other places, but `defaultRole` takes time to be defined, therefore `roleField` can get initialized with an empty value. 

The way that I got to fix it was by introducing this effect here which runs once the default role gets updated, sets the value, and doesn't execute anymore once the field has a value.